src-exe/LicenseReport.hs

```diff
@@ -40,7 +40,7 @@ import           Prelude.Compat
 
 import Cabal.Config (readConfig, Config (..), cfgRepoIndex, hackageHaskellOrg)
 
-#if MIN_VERSION_Cabal(3,2,0)
+#if MIN_VERSION_Cabal_syntax(3,2,0)
```

Comment:
:information_source: Please double check if the actual version bound (3.2.0) still makes sense after switching to `Cabal-syntax`

Reply:
The version guards can be removed, as we always require at least `Cabal-syntax >=3.8.1.0`

Reply:
Right